### ISSUE TITLE
[Feature] Add support for editing submission in submission details panel

### ISSUE THREAD

**Issue by amysorto** [google/ground-platform#1280]

**Comment by kenstershiro** [comment]
From @jabramowitz5 this is important - if we know someone made an error (e.g. multiple saves that they weren't sure if it worked), or if we've added test surveys (potentially a separate ticket for the test/draft use case)

**Comment by vittorino** [comment]
Deleting a submission from within its panel:
![Image](https://github.com/user-attachments/assets/81c565ba-061f-4c98-bcab-2bf9c2e8da8c)

Confirmation:
![Image](https://github.com/user-attachments/assets/c2543f75-b5d1-4ab1-b3d9-8ef0734438a4)

**Comment by vittorino** [comment]
Deleting multiple submissions:
![Image](https://github.com/user-attachments/assets/8eabd7de-9e13-4eb1-8fac-cdb4b3261771)

Selection:
![Image](https://github.com/user-attachments/assets/452a2d1d-b5d1-4006-9176-84bf4374f02a)

Action:
![Image](https://github.com/user-attachments/assets/a050aa4a-57fb-4020-b647-a31c09e19ea4)

Confirmation:
![Image](https://github.com/user-attachments/assets/9fdee9d5-6ae0-4cc3-8933-2669259d1f10)

**Comment by vittorino** [comment]
@gino-m @amegantz FYI

**Comment by gino-m** [comment]
> Deleting a submission from within its panel: ![Image](https://github.com/user-attachments/assets/81c565ba-061f-4c98-bcab-2bf9c2e8da8c)
> 
> Confirmation: ![Image](https://github.com/user-attachments/assets/c2543f75-b5d1-4ab1-b3d9-8ef0734438a4)

So far we've only been asked to remove one at a time, so this seems like the right pattern. @n-clinton @lecrabe @jabramowitz5 Can you confirm?

**Comment by jabramowitz5** [comment]
@gino-m @vittorino Yes, I think going with the option to remove one at a time is the best way forward. This will help to ensure survey organizers do not accidentally delete "good" entries since it is a permanent action. this is a process we want to make sure survey organizers are very carefully carrying out.

**Comment by jo-spek** [comment]
This has come up as feature request twice in the past month.

@kenstershiro

**Comment by kenstershiro** [comment]
@rfontanarosa  to discuss feasibility with @gino-m 

- delete submissions (UX designs provided)
- edit answers to survey questions
- edit LOIs (within Ground web platform, e.g. edit vertices of a polygon, or export to a an external tool and reimport) - see #2212

**Comment by kenstershiro** [comment]
delete submissions (UX designs provided) is considered the top priority, and no blockers or dependencies. @jo-spek please create a separate issue and add to next iteration for @rfontanarosa
